packages/fxa-settings/README.md

```diff
@@ -203,16 +205,21 @@ const LogoImage = () => <div class="logo" role="img" aria-label="logo"></div>;
 
 ## Testing
 
-This package uses [Jest](https://jestjs.io/) to test its code. By default `npm test` will test all JS files under `src/`.
+This package uses [Jest](https://jestjs.io/) to test its code. By default `yarn test` will test all JS files under `src/`.
 
 Test specific tests with the following commands:
 
 ```bash
 # Test for the component AlertBar
```

Comment:
This must have been me when I originally wrote this, but this comment should be AppLayout, not AlertBar